src/containers/shared/components/Transaction/NFTokenAcceptOffer/Simple.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { SimpleRow } from '../SimpleRow'
+import { Amount } from '../../Amount'
+import { NFTokenAcceptOfferInstructions } from './types'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps<NFTokenAcceptOfferInstructions>) => {
+  const { acceptedOfferIDs, amount, tokenID, seller, buyer } =
+    data?.instructions
+  const { t } = useTranslation()
+
+  return (
+    <>
+      {acceptedOfferIDs.map((offer) => (
+        <SimpleRow label={t('offer_index')} className="dt" data-test="offer-id">
+          {offer}
+        </SimpleRow>
+      ))}
+      {amount && seller && buyer && tokenID && (
+        <>
+          <SimpleRow label={t('seller')} className="account" data-test="seller">
+            <Account account={seller} />
+          </SimpleRow>
+          <SimpleRow label={t('buyer')} className="account" data-test="buyer">
+            <Account account={buyer} />
+          </SimpleRow>
+          <SimpleRow label={t('token_id')} className="dt" data-test="token-id">
```

Comment:
I think I told you incorrectly to use dt className on SimpleRow.  The css says it should be inside `.row` to have styles applied.  The name dt is not great (I realize it has been around for a while).  Originally it was used for showing `DestinationTag`s.  Maybe the class name should be changed to `long`.  And not set the different gray color. @mvadari Thoughts?

Reply:
For this PR, what should be changed?

Reply:
I will create a new ticket for the `dt` thing

Reply:
You should still remove the "account" className

Reply:
Created #361